### ISSUE TITLE
currentStencilSize was mentioned but never processed

### ISSUE THREAD

**Issue by a0s** [Norserium/vue-advanced-cropper#281]
During rewriting vue-advanced-cropper to better typed SFC i found mentions of `currentStencilSize`, which is not supported in `algorithms.calculateStencilSize`

**Comment by Norserium** [comment]
@a0s, you are right, but what's exactly do you want to improve by this rewriting?
